pkg/virtctl/create/params/params_test.go

```diff
@@ -0,0 +1,262 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+package params_test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"kubevirt.io/kubevirt/pkg/pointer"
+	"kubevirt.io/kubevirt/pkg/virtctl/create/params"
+)
+
+var _ = Describe("params", func() {
+	const (
+		testParam = "testParam"
+		flagName  = "testFlag"
+	)
+
+	Context("NotFoundError", func() {
+		It("should have correct message", func() {
+			err := params.NotFoundError{Name: testParam}
+			Expect(err.Error()).To(Equal(testParam + " must be specified"))
+		})
+
+		DescribeTable("should detect equal errors when passed as", func(target error) {
+			err := params.NotFoundError{Name: testParam}
+			Expect(err.Is(target)).To(BeTrue())
+		},
+			Entry("value", params.NotFoundError{Name: testParam}),
+			Entry("pointer", pointer.P(params.NotFoundError{Name: testParam})),
+		)
+
+		It("should detect unequal errors", func() {
+			err := params.NotFoundError{Name: testParam}
+			Expect(err.Is(os.ErrNotExist)).To(BeFalse())
+		})
+	})
+
+	Context("FlagErr", func() {
+		const (
+			format = "My wrapped test error: %s"
+			value  = "testvalue"
+		)
+
+		It("should have correct message", func() {
+			err := params.FlagErr(flagName, format, value)
+			Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("failed to parse \"--%s\" flag: ", flagName))))
```

Comment:
nit - Would you mind sharing these strings as public constants from the params package?